src/command_modules/azure-cli-consumption/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.3.1
++++++
+* Added commands `budget`.
```

Comment:
Change to clearer description. Currently, this gives very little information.
Here are some examples from previous releases of the module.
https://docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest#consumption
https://docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest#consumption-1